includes/models/model.llms.question.php

```diff
@@ -356,12 +356,12 @@ public function get_correct_choice() {
 	/**
 	 * Get the question text (title)
 	 *
-	 * @since 3.16.0
+	 * @since [version]
```

Comment:
Can you fix this method docblock? the first since must stay because it's when the method has been created, then you need another since for the changes you introduced now.
Also this method misses the `@param`

Reply:
fixed.